tools/sort-fails.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash +x
+
+# Copyright 2015 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+INPUT_PATH="$1"
+VERSION=$(git log -n 1 --pretty=format:"%H")
+SYSTEM=$(uname -a)
+TMP_DIR="afl-testing"
+USAGE="Usage:\n\t./tools/sort-fails.sh PATH_TO_DIR_WITH_FAILS"
+
+if [ "$#" -ne 1 ]
+then
+  echo -e "${USAGE}";
+  echo "Argument number mismatch...";
+	exit 1;
+fi
+
+rm $TMP_DIR -r; mkdir $TMP_DIR;
```

Comment:
Could you, please, change temporary directory construction to `mktemp -d` like in `tools/prerequsites.sh` script? Otherwise, this command could eventually remove some necessary data.